.github/workflows/build_and_test.yml

```diff
@@ -75,33 +81,38 @@ jobs:
       - name: Build OSX Client
         run: npx gulp build --platform=osx
 
-  # TODO: setup fastlane for iOS
-  
-  # ios:
-  #   name: iOS
-  #   runs-on: macos-11
-  #   steps:
-  #     - name: Checkout
-  #       uses: actions/checkout@v2.3.4
+  ios:
+    name: iOS
+    runs-on: macos-11
+    timeout-minutes: 10
+    env:
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
```

Comment:
I don't see the secret in the project settings. Where is this coming from?

Reply:
Also you can do `environment: <env name>` and get all the env vars set, instead of specifying each here.

Reply:
It seems `environment` is tied 1:1 with deployments - just checking to see if that's what you wanted. We're currently not deploying anything so it feels like a misnomer, slightly

Reply:
also the variables didn't seem to populate - https://github.com/Jigsaw-Code/outline-client/runs/4446103793?check_suite_focus=true#step:6:299

secrets here:
![Screen Shot 2021-12-07 at 10 45 54 AM](https://user-images.githubusercontent.com/3759828/145060818-c2a3d0c0-887c-4abe-9055-7fadcc0591bd.png)

I wonder if i have to pass them in via `env` anyway

Reply:
It seems you need to explicitly create the env vars and use the `secrets` object: https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
Got it, so revert that last change, basically 😝